### PR TITLE
[FIX] web: `ListRenderer`, invalid cell border-color

### PR DESCRIPTION
**Pull request by stefanorigano**
task-2984169

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by stefanorigano** [comment]
Hello @aab-odoo , the `ListRenderer` issue you reported + some fixes on the same component asked by fp.

Thanks

**Review by aab-odoo — COMMENTED** [review]
Hello @stefanorigano 

Could you check again because I recently removed the padding from SelectCreateDialog (dialogs with list views), like in legacy?

File: addons/web/static/src/core/dialog/dialog.xml [hunk 1]
```diff
@@ -30,7 +30,7 @@
                                 </button>
                             </t>
                         </footer>
-                        <main class="modal-body" t-att-class="{'p-0': !props.withBodyPadding}">
+                        <main class="modal-body" t-att-class="props.withBodyPadding ? 'hasBodyPadding' : 'p-0'">
```
Comment (aab-odoo) [inline addons/web/static/src/core/dialog/dialog.xml:33]: I'm not sure about our className conventions, but I don't think this one follows them.

**Review by aab-odoo — APPROVED** [review]

**Comment by stefanorigano** [comment]
Hello @aab-odoo , retarget on 16.0.

Since https://github.com/odoo/odoo/pull/100580 we enforce the old negative-margins override system, I think it's not worth to change it for now.

I keep the border-color fix only.

**Comment by aab-odoo** [comment]
Hello @stefanorigano . I think you have to create another branch, prefixed by 16.0- for the ci/template to be green.

**Comment by stefanorigano** [comment]
Closed in favour of https://github.com/odoo/odoo/pull/101112

**Pull request closed without merging**